### DISCUSSION TITLE
🚀 Feature Request: wrangler pages project create --git-repo

### DISCUSSION THREAD

**Discussion by Zundrium** [cloudflare/workers-sdk#13480] [Feature Requests]
### Describe the solution

![image](https://github.com/user-attachments/assets/933697cd-fec2-4e22-b52a-963a30828401)

As far as I can see, wrangler only supports creating pages projects with folders.

I would like to see a command where I can create a pages project based on an existing git project like the tool above.

**Comment by CarmenPopoviciu** [comment]
cc @aninibread ... not sure if this is something we'd want to support for Pages...or Workers, now that we have Workers CI

**Comment by Zundrium** [comment]
Thank you for taking a look at it!

It's just that this is the only step missing for me to deploy pages projects automatically.
